diff --git a/setup.py b/setup.py
index 858bb39..4ed1678 100644
--- a/setup.py
+++ b/setup.py
@@ -783,7 +783,6 @@ options =  dict(
   url = 'https://github.com/vtraag/louvain',
 
   use_scm_version=True,
-  setup_requires=['setuptools_scm'],
 
   author = 'V.A. Traag',
   author_email = 'vincent@traag.net',
@@ -793,7 +792,6 @@ options =  dict(
   package_dir = {'louvain': 'src'},
   packages = ['louvain'],
   ext_modules = [louvain_ext],
-  install_requires = ['python-igraph >= 0.8.0'],
   platforms="ALL",
   keywords=[
     'graph',
